zeppelin-client/pom.xml

```diff
@@ -48,7 +48,7 @@
 
     <dependency>
       <groupId>org.eclipse.jetty.websocket</groupId>
-      <artifactId>websocket-client</artifactId>
+      <artifactId>websocket-jetty-client</artifactId>
```

Comment:
Does zeppelin-client use jetty client for websocket, not jakarta?

Reply:
Thank you for clarifying it. I think it should be fine for now. I just asked implementation detail.